docs/source/conceptual/visual_language_models.md

```diff
@@ -0,0 +1,224 @@
+# Vision Language Models (VLM)
```

Comment:
```suggestion
# Vision Language Model Inference in TGI
```
In this guide you don't talk about serving but rather inference so better to be specific

Reply:
resolved